util/metrics.go

```diff
@@ -0,0 +1,140 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
```

Comment:
I wouldn't put more stuff in `util` as it will become a trash can eventually. What about another package called `metrics`?

Reply:
Sure! I will put this code in a new metrics package.
I was thinking in the last changes that this kind of naming `utils.ServiceInfo` was not meaningful. So, better in its own package.